src/pages/basic-management/employee/list/Index.js

```diff
@@ -0,0 +1,473 @@
+import React, { PureComponent, Fragment } from 'react';
+import router from 'umi/router';
+import { connect } from 'dva';
+import {
+    List,
+    Card,
+    Row,
+    Col,
+    Radio,
+    Input,
+    InputNumber,
+    Progress,
+    Button,
+    Icon,
+    Dropdown,
+    Menu,
+    Avatar,
+    Modal,
+    Form,
+    DatePicker,
+    Select,
+    Badge,
+    Cascader,
+    Popconfirm,
+    Divider,
+} from 'antd';
+const { MonthPicker, RangePicker, WeekPicker } = DatePicker;
+import StandardTable from '@/components/StandardTable';
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import styles from '../../../List/TableList.less';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+const getValue = obj =>
+    Object.keys(obj)
+        .map(key => obj[key])
+        .join(',');
+const statusMap = ['success', 'error'];
+const status = ['在职', '离职'];
+
+@connect(({ employee, depart, loading }) => ({
+    employee,
+    depart,
+    loading: loading.models.employee,
+}))
+@Form.create()
+class employeeList extends PureComponent {
+    state = {
```

Comment:
Codacy found an issue: [State initialization should be in a constructor](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)